test/integration/targets/nxos_smoke/tasks/nxapi.yaml

```diff
@@ -25,9 +25,3 @@
   with_items: "{{ test_items }}"
   loop_control:
     loop_var: test_case_to_run
-
-- name: run test cases (connection=local)
-  include: "{{ test_case_to_run }} ansible_connection=local connection={{ nxapi }}"
```

Comment:
We would still want to keep running nxapi with local for at least one target. Let's do it for smoke.